### PR TITLE
Adding pitest-cucumber-plugin to related tools

### LINKED ISSUE #245: Add pitest-cucumber-plugin to related tools
Add [pitest-cucumber-plugin](https://github.com/alexvictoor/pitest-cucumber-plugin) to [Related tools](http://docs.cucumber.io/tools/related-tools/) page.
Note: Tools are listed in alphabetical order.

### PR DESCRIPTION
**Pull request by TomerPacific**
Fixes #245

### CODE REVIEW

File: content/tools/related-tools.md [hunk 1]
```diff
@@ -60,6 +60,9 @@ People have written tools on top of Cucumber. These are the known third-party to
...
 | [Pickle](https://github.com/ianwhite/pickle/tree/master)                                                        | Easy model creation/reference in Cucumber - optionally leveraging your factories/blueprints                                                                                                                                                                                                  |
 | [Pickler](https://github.com/tpope/pickler/tree/master)                                                         | Synchronize user stories in [Pivotal Tracker](http://www.pivotaltracker.com/) with Cucumber Features.                                                                                                                                                                                        |
+| [Pitest-Cucumber-Plugin](https://github.com/alexvictoor/pitest-cucumber-plugin)                                 | Cucumber intergration for mutation testing tool PIT
+
+  |
```
Comment (olleolleolle) [inline content/tools/related-tools.md]: Something about this whitespace made the following lines weird. Try removing newlines "inside" the table cell.

Comment (TomerPacific) [reply content/tools/related-tools.md]: @olleolleolle , noticed it as well and fixed it

File: content/tools/related-tools.md [hunk 2]
```diff
@@ -60,6 +60,7 @@ People have written tools on top of Cucumber. These are the known third-party to
...
 | [Pickle](https://github.com/ianwhite/pickle/tree/master)                                                        | Easy model creation/reference in Cucumber - optionally leveraging your factories/blueprints                                                                                                                                                                                                  |
 | [Pickler](https://github.com/tpope/pickler/tree/master)                                                         | Synchronize user stories in [Pivotal Tracker](http://www.pivotaltracker.com/) with Cucumber Features.                                                                                                                                                                                        |
+| [Pitest-Cucumber-Plugin](https://github.com/alexvictoor/pitest-cucumber-plugin)                                 | Cucumber intergration for mutation testing tool PIT                                                                                         |
```
Comment (olleolleolle) [inline content/tools/related-tools.md]:
```suggestion
| [Pitest-Cucumber-Plugin](https://github.com/alexvictoor/pitest-cucumber-plugin)                                 | Cucumber integration for mutation testing tool PIT                                                                                         |
```

**Comment by olleolleolle** [comment]
You can also change the _Description_ text to include `Fixes #245`, which then becomes a "magic action word" - the referenced Issue will be closed on this PR being merged.

**Review by olleolleolle — APPROVED** [review]
Thanks for working on this! :hearts: :yellow_heart: :green_heart: :blue_heart: :purple_heart:

**Comment by aslakhellesoy** [comment]
Hi @TomerPacific,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Comment by mlvandijk** [comment]
Awesome! Thanks @TomerPacific and @olleolleolle!

**Pull request merged**